build.py

```diff
@@ -30,10 +31,13 @@
 DEFAULT_CONFIG = "etc/kapacitor/kapacitor.conf"
 PREINST_SCRIPT = None
 
+# Default AWS S3 bucket for uploads
+DEFAULT_BUCKET = "kapacitor"
```

Comment:
I like the idea of using a kapacitor specific bucket. I'll just make sure the next release we update the bucket path. Thanks